packages/core/testing/cable/index.js

```diff
@@ -0,0 +1,29 @@
+export class TestCable {
+  constructor() {
+    this.outgoing = []
+    this.channels = {}
+  }
+
+  async subscribe(channel) {
+    channel.connecting(this)
+
+    this.channels[channel.identifier] = channel
+    channel.connected(channel.identifier)
+
+    return Promise.resolve(channel.identifier)
+  }
+
+  async perform(identifier, action, payload = {}) {
```

Comment:
I think, we should check for `identifier` existence here and throw an error if id is absent (like here https://github.com/anycable/anycable-client/blob/0f22fa8cfc3223e2b8d54cbc005a8d650dd76e56/packages/core/cable/index.js#L254)

Reply:
Added error throwing for non existing identifier